client/src/Components/Card.js

```diff
@@ -27,15 +28,16 @@ const Card = (props) => {
     const requestOptions = {
       method: "POST",
       headers: { "Content-Type": "application/json" },
-      body: JSON.stringify({ userId: user._id, itemId: id }),
+      body: JSON.stringify({ itemId: id }),
     };
     try {
-      const response = await (await fetch("/addToCart", requestOptions)).json();
-      if (response) {
+      const result = await (await fetch("/addToCart", requestOptions)).json();
```

Comment:
don't use nested await. Either use await in different vars or use then-catch statements